packages/design-system/src/components/Table/TableCell.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableCell = ({
+  children,
+  className,
+  data,
+  stackedTitle,
+  stackedClassName,
+  type,
+  ...attributeOptions
+}) => {
+  const classes = classNames(
+    'ds-c-table__cell',
+    className,
+    type === 'numeric' ? 'ds-c-table__cell--numeric' : null
+  );
+
+  const stackedClasses = classNames(
+    'ds-c-table--stacked__col-header',
+    'ds-u-font-weight--bold',
+    stackedClassName
+  );
+
+  const renderStackedTitle = () => {
+    const isValidStackedTitle = stackedTitle && stackedTitle.length > 0;
+
+    return (
+      { isValidStackedTitle } && (
+        <span aria-hidden="true" className={stackedClasses}>
+          {stackedTitle}
+        </span>
+      )
+    );
+  };
+
+  return (
+    <td className={classes} role="cell" {...attributeOptions}>
+      {renderStackedTitle()}
+      {data}
+      {children}
+    </td>
+  );
+};
+
+TableCell.defaultProps = {
+  className: '',
+  stackedClassName: '',
+  type: 'text',
+};
+
+TableCell.propTypes = {
+  /**
+   * The table cell contents.
+   */
+  children: PropTypes.node,
+  /**
+   * Additional classes to be added to the table cell element.
+   */
+  className: PropTypes.string,
+  /**
+   * The table cell data.
+   */
+  data: PropTypes.node.isRequired,
+  /**
+   * The `headers` attribute contains a list of `id` attributes of the associated data. If there is more than one id, they are separated by spaces.
+   */
+  headers: PropTypes.string,
```

Comment:
Is this prop used in this component?

Reply:
For complex table with multi-headers table or irregular header columns, users need to assign an `id` to each column heading and tie that `id` to the respective cell via the Cell `headers`
see here for examples: https://codepen.io/tpierce_402/pen/YzzqwmZ

Reply:
Right I understand that, I was asking if this `prop` was being used. I see now that it's inside `attributeOptions`. I think in this case I would suggest explicitly define `headers` instead of using `attributeOptions`.